pkg/config/config.go

```diff
@@ -217,8 +217,16 @@ type OS struct {
 }
 
 type ExternalStorageConfig struct {
-	Enabled         bool         `json:"enabled,omitempty"`
-	MultiPathConfig []DiskConfig `json:"multiPathConfig,omitempty"`
+	Enabled bool `json:"enabled,omitempty"`
+
+	MultiPathConfig MultiPathConfig `json:"multiPathConfig,omitempty"`
+}
+
+type MultiPathConfig struct {
+	Blacklist     []DiskConfig `json:"blacklist,omitempty"`
+	BlackListWwid []string     `json:"blackListWwid,omitempty"`
```

Comment:
In the field names and json above, one is `Blacklist` and `blacklist` (lower case 'l') the other is `BlackListWwid` and `blackListWwid` (capital `L`). These should probably be unified.